src/win/wgl_disp.c

```diff
@@ -967,7 +968,9 @@ static bool create_display_internals(ALLEGRO_DISPLAY_WGL *wgl_disp)
    if ((disp->flags & ALLEGRO_OPENGL_3_0) || major != 0) {
       if (major == 0)
          major = 3;
-      bool fc = (disp->flags & ALLEGRO_OPENGL_FORWARD_COMPATIBLE) != 0;
+	  if ((disp->flags & ALLEGRO_OPENGL_FORWARD_COMPATIBLE) != 0)
```

Comment:
that != 0 there is a bit unnecessary if we are introducing a new if() statement here...

Reply:
You're right, fixed in the next commit. Thanks.